src/main/resources/application.yaml

```diff
@@ -4,19 +4,7 @@ server:
 spring:
   application:
     name: API-GATEWAY
-  cloud:
-    gateway:
-      routes:
-        - id: STUDENT-SERVICE
-          uri: lb://STUDENT-SERVICE
-          predicates:
-            - Path=/students/**
-        - id: COURSE-SERVICE
-          uri: lb://COURSE-SERVICE
-          predicates:
-            - Path=/courses/**
 
-#lb://STUDENT-SERVICE = load balanced://application.name
 eureka:
```

Comment:
You probably no longer need these eureka properties.

Reply:
Fixes will be applied in DTOs branch's PR